### ISSUE TITLE
Proxy Mode

### ISSUE THREAD

**Issue by JosephSamela** [MrChuckomo/poddycast#29]
Love the app! Having a little trouble with proxy mode.

I have `HTTP_PROXY` and `HTTPS_PROXY` set as environment variables and `Proxy Mode` checked in settings. My proxy is working with other applications just not Poddycast. I'm using Windows 7.

![untitled](https://user-images.githubusercontent.com/16111451/39139993-5eb02f9a-46f1-11e8-8482-a62c417baf46.png)

Entering a search term logs errors to console:
```
Problem with request: connect ECONNREFUSED 143.102.42.55:8080
```

Any tips for getting this setup? Thanks!

![image](https://user-images.githubusercontent.com/16111451/39138825-a168879a-46ee-11e8-8f35-12569d28ea10.png)

**Comment by MrChuckomo** [comment]
Sorry to hear that. Looks like a bug. 
I guess I have to investigate this and change the code.

**Comment by xmha97** [comment]
+1, also I need proxy settings.

**Comment by EthanGrahn** [comment]
I've started taking a look into this. The request library that we are using says that it supports proxy configuration, but I have been unable to get it to work so far. I will keep digging into it, but if any other developers look into it and find progress, please let me know.

**Comment by EthanGrahn** [comment]
This will be closed once #102 is merged in but if you still run into issues with the proxy in that version, let me know here so we can take another look.
